src/app-layout/visual-refresh/trigger-button.tsx

```diff
@@ -33,37 +52,69 @@ function TriggerButton(
     ariaControls,
     onClick,
     testId,
+    disabled = false,
     badge,
     selected = false,
     highContrastHeader,
   }: TriggerButtonProps,
   ref: React.Ref<ButtonProps.Ref>
 ) {
+  const { isMobile } = useAppLayoutInternals();
+
   return (
-    <div className={clsx(styles['trigger-wrapper'], !highContrastHeader && styles['remove-high-contrast-header'])}>
-      <button
-        aria-expanded={ariaExpanded}
-        aria-controls={ariaControls}
-        aria-haspopup={true}
-        aria-label={ariaLabel}
-        className={clsx(
-          styles.trigger,
-          {
-            [styles.selected]: selected,
-            [styles.badge]: badge,
-          },
-          className
-        )}
-        onClick={onClick}
-        ref={ref as React.Ref<HTMLButtonElement>}
-        type="button"
-        data-testid={testId}
-      >
-        <span className={clsx(badge && styles['trigger-badge-wrapper'])}>
-          <Icon name={iconName} svg={iconSvg} />
-        </span>
-      </button>
-      {badge && <div className={styles.dot} />}
+    <div
+      className={clsx(styles['trigger-wrapper'], !highContrastHeader && styles['remove-high-contrast-header'])}
+      data-testid={`${testId ? `${testId}-wrapper` : 'awsui-app-layout-trigger-wrapper'}`}
```

Comment:
We typically query DOM selectors in our test utils/test cases using CSS classes to avoid potential name collisions with the consumers' code

Reply:
ok.  I can add classes and change this